activity/inference/framework/tf/load.go

```diff
@@ -7,19 +7,29 @@ import (
 
 	models "github.com/TIBCOSoftware/flogo-contrib/activity/inference/model"
 	tfpb "github.com/TIBCOSoftware/flogo-contrib/activity/inference/tensorflow/tensorflow/core/protobuf"
+	"github.com/TIBCOSoftware/flogo-lib/core/activity"
 	"github.com/golang/protobuf/proto"
 	tf "github.com/tensorflow/tensorflow/tensorflow/go"
 )
 
 // Load implements the backend framework specifics for loading a saved model
-func (a *TensorflowModel) Load(modelPath string, modelFile string, model *models.Model) (err error) {
+func (a *TensorflowModel) Load(modelPath string, modelFile string, model *models.Model, context activity.Context) (err error) {
 	var meta models.Metadata
```

Comment:
may rather than passing around activity.context we should pass in just sig def and tag into the load. We can update the model.metadata and add the sigdef and tag to the metadata inside load, that way we dont need to also pass to parseProto, etc...

Reply:
Done.  I created a struct in the model package to carry these flags.